src/libasr/intrinsic_func_registry_util_gen.py

```diff
@@ -702,6 +705,9 @@ def add_create_func_return_src(func_name):
         for _i in range(no_of_args):
             src += indent * 3 + f"args_values.push_back(al, expr_value(m_args[{_i}]));\n"
         src += indent * 3 +     f"m_value = eval_{func_name}(al, loc, return_type, args_values, diag);\n"
+        src += indent * 2 + "if (diag.has_error()) {\n"
+        src += indent * 3 +     f"return nullptr;\n"
+        src += indent * 2 + "}\n"
```

Comment:
It looks like this should be:
```suggestion
        src += indent * 3 +     "if (diag.has_error()) {\n"
        src += indent * 4 +         f"return nullptr;\n"
        src += indent * 3 +     "}\n"
```

Reply:
Done